priv/repo/migrations/20210216142615_add_vehicle_adherence_color_to_user_setting.exs

```diff
@@ -11,6 +11,6 @@ defmodule Skate.Repo.Migrations.AddVehicleAdherenceColorToUserSetting do
       add(:vehicle_adherence_colors, :vehicle_adherence_colors, default: "early_red")
     end
 
-    execute("UPDATE user_settings SET vehicle_adherence_colors = 'early_red'", fn -> end)
+    execute("UPDATE user_settings SET vehicle_adherence_colors = 'early_red'", fn -> nil end)
```

Comment:
Extraneous change to a migration that already ran (did `mix format` do this maybe?)

Reply:
I did this on purpose to stop a warning I was getting every time I ran. I think my linter is maybe set to stricter standards than yours, we should coordinate on that.

Reply:
Oh, interesting. Yeah, we should figure out what's up with that at some point.